std/utf.d

```diff
@@ -4278,9 +4278,9 @@ private int impureVariable;
  *      Does not use GC if `useReplacementDchar` is set to `UseReplacementDchar.no`
  *
  * Returns:
- *      A forward range if `R` is a range and not auto-decodable, as defined by
+ *      A bidirectional range if `R` is a range and not auto-decodable, as defined by
  *      $(REF isAutodecodableString, std, traits), and if the base range is
- *      also a forward range.
+ *      also a bidirectional range.
```

Comment:
You are omitting the fact that if the underlying range is a forward range, a forward range is returned.

Reply:
@RazvanN7 fixed it.